test/vcr_python_wrapper.py

```diff
@@ -7,23 +7,6 @@
 import re
 
 
-# We need our own json level2 matcher, because, python2 and python3 do not save
-# dictionaries in the same order
-def body_json_l2_matcher(r1, r2):
-    if r1.headers.get('content-type') == 'application/json' and r2.headers.get('content-type') == 'application/json':
-        if r1.body is None or r2.body is None:
-            return r1.body == r2.body
-        body1 = json.loads(r1.body.decode('utf8'))
-        body2 = json.loads(r2.body.decode('utf8'))
-        if 'search' in body1:
-            body1['search'] = ','.join(sorted(re.findall(r'([^=,]*="(?:[^"]|\\")*")', body1['search'])))
-        if 'search' in body2:
-            body2['search'] = ','.join(sorted(re.findall(r'([^=,]*="(?:[^"]|\\")*")', body2['search'])))
-        return body1 == body2
-    else:
-        return r1.body == r2.body
-
-
```

Comment:
Did they change the behavior of vcrpy here?

Reply:
I think it may have been https://github.com/kevin1024/vcrpy/pull/340/files#diff-dc8efcefc2dd30550706019d468fa580R1